change/@ni-nimble-components-4c6ed2fc-5a8c-4af1-bc8d-d145422fca42.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

It doesn't look like any code is actually changing in this PR, so the type should be none.
```suggestion
  "type": "none",
```